server/testutils/addResponseToApplication.ts

```diff
@@ -2,15 +2,31 @@ import { ApprovedPremisesApplication as Application } from '../@types/shared'
 
 export const addResponseToApplication = (
```

Comment:
I wonder if this should be broken into two and have an `addResponsesToApplication` that accepts an object, to save the duplication and conditionals.

Reply:
Yeah, seems reasonable